acts

```diff
@@ -114,7 +114,7 @@ else
     # There's no yearly backup
     archivetype="yearly"
 fi
-log_verbose "Creating $archivetype backup"
+log_verbose "at=start type=$archivetype"
```

Comment:
Reverting this as well. It's somewhat redundant with log messages below that have more context.